lib/helper/shtml.js

```diff
@@ -41,9 +40,12 @@ module.exports = function shtml(val) {
         if (value[0] === '/' || value[0] === '#') {
           return;
         }
-
         const hostname = urlparse(value).hostname;
-        // 既不在内置白名单，也不再 config.helper.shtml.domainWhiteList 中，则过滤掉
+        // neither in inner domain white list nor in config.helper.shtml.domainWhiteList will be filtered
+        // config.helper.shtml.domainWhiteList has been deprecated
+        if (shtmlConfig.domainWhiteList && shtmlConfig.domainWhiteList.length !== 0) {
```

Comment:
You can delete domainWhiteList in config.default.js, then only test `shtmlConfig.domainWhiteList`